com.unity.render-pipelines.universal/Runtime/ShaderUtils.cs

```diff
@@ -3,22 +3,72 @@
 
 namespace UnityEngine.Rendering.Universal
 {
+    /// <summary>
+    /// Options to get a shader path to URP shaders when calling ShaderUtils.GetShaderGUID();
+    /// <see cref="ShaderUtils"/>.
+    /// </summary>
     public enum ShaderPathID
     {
+        /// <summary>
+        /// Use this for URP Lit shader.
```

Comment:
Is ComplexLit missing from this list? Should it be here?
Anyway ComplexLit is a superset of Lit for advanced effects that would need additional GBuffer storage. I is always forward rendered and it's used as deferred fallback (for those effects).